src/ClientProtos/capture_data.proto

```diff
@@ -164,6 +167,36 @@ message TimerInfo {
   uint64 timeline_hash = 11;
   repeated uint64 registers = 12;
   Color color = 13;
+  uint64 group_id = 14;
+  uint64 id = 15;
+  uint64 address_in_function = 16;
+  string name = 17;
+}
+
+message ApiStringEvent {
```

Comment:
these are not saved anywhere, maybe we do not need messages here just structs?

Reply:
The Idea was to just be consistent for the rest until we have the CaptureDataProvider.

Reply:
Also, we can probably use this as an impulse to get rid of the serialization code.

Reply:
Both fair points...